io/disk/dbench.py

```diff
@@ -0,0 +1,71 @@
+import os
+import re
+import multiprocessing
+import json
+
+from avocado import Test
+from avocado import main
+from avocado.utils import archive
+from avocado.utils import process
+from avocado.utils import build
+
+
+class Dbench(Test):
+
+    """
+    Dbench is a tool to generate I/O workloads to either a filesystem or to a
+    networked CIFS or NFS server.
+    Dbench is a utility to benchmark a system based on client workload
+    profiles.
+    """
+
+    def setUp(self):
+        '''
+        Build Dbench
+        Source:
+        http://samba.org/ftp/tridge/dbench/dbench-3.04.tar.gz
+        '''
+
+        try:
+            process.run('which gcc')
+        except:
+            self.error('gcc is required by this job and is not available on'
+                       'the system')
+
+        self.results = []
+        data_dir = os.path.abspath(self.datadir)
+        tarball = self.fetch_asset(
+            'http://samba.org/ftp/tridge/dbench/dbench-3.04.tar.gz')
+        archive.extract(tarball, self.srcdir)
+        cb_version = os.path.basename(tarball.split('.tar.')[0])
+        self.srcdir = os.path.join(self.srcdir, cb_version)
+        os.chdir(self.srcdir)
+        patch = self.params.get('patch', default='dbench_startup.patch')
+        process.system('patch -p1 < %s' % data_dir + '/' + patch, shell=True)
```

Comment:
The practical difference between `process.system()` and `process.run()` is the first returns the command exit code and the last returns the `CmdResult` class object.
Since you're not using the functions returns, you could use the same function for both cases so you don't confuse code readers.